src/api/procedures/consumeJoinIdentityAuthorization.ts

```diff
@@ -67,26 +69,33 @@ export async function prepareConsumeJoinIdentityAuthorization(
 /**
  * @hidden
  */
-export async function isAuthorized(
+export async function getAuthorization(
   this: Procedure<ConsumeJoinIdentityAuthorizationParams>,
   { authRequest, accept }: ConsumeJoinIdentityAuthorizationParams
-): Promise<boolean> {
+): Promise<ProcedureAuthorization> {
   const { target, issuer } = authRequest;
   const { context } = this;
 
   let condition;
   let did: string | undefined;
   const fetchDid = async (): Promise<string> => getDid(did, context);
 
+  let transactions: TxTag[] = [];
+
+  let paidByThirdParty = false;
+
   if (target instanceof Account) {
     const { address } = context.getCurrentAccount();
     condition = address === target.address;
+    paidByThirdParty = condition;
```

Comment:
I think `transactions = [TxTags.identity.JoinIdentityAsKey];` is missing here

Reply:
No, that would always fail since the account can't have any permissions if it's not affiliated to any Identity